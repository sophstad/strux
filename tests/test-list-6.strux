int main() {
  LinkedList::int l = new LinkedList::int();
  l.add(6000000);
  l.add(0);
  l.add(-326);
  l.add(7 + 89);
  l.show();
  return 0;
}
