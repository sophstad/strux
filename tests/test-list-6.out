[ 6000000 ] -> [ 0 ] -> [ -326 ] -> [ 96 ]
